dev_guide/deployments.adoc

```diff
@@ -138,16 +170,28 @@ $ osc describe dc <deployment_config>
 ----
 
 == Canceling a Deployment
+You might want to cancel a deployment if:
+
+* A deployment was initiated incorrectly, and you want to cancel it before it can do further damage.
+* There is a security fix you need to apply, and you do not want to wait for an ongoing deployment to complete.
+* A deployment or hook implementation is stuck after running for too long, and you cannot initiate a deployment when there is a deployment already running.
+
+[NOTE]
+====
+By default, the deployment process is restricted to 6 hours. If a deployment is
+still in progress after 6 hours, then the containers for the deployment pod and
+any deployment hook pods are all deleted, and these pods are placed in the
+Failed state.
+====
 
-To cancel a running deployment:
+Cancel a running or stuck deployment with the following command:
```

Comment:
More consistent with the other examples would be:

"To cancel a running or stuck deployment:"